docs/END_USER_DOCUMENT.md

```diff
@@ -612,6 +614,23 @@ MoccaResilience4j moccaResilience2 = new MoccaResilience4j.Builder()
     .build();
 ```
 
+#### 6.5.2 Adding fallback to your client
```

Comment:
Please remove this section, it is not necessary.
Instead, just add `.fallback(fallback)` in the example under section 6.5.1.

Also, notice the paragraph below should be under section 6.5.1. You removed it (probably accidentally).

```
Notice the instantiation and configuration of `rateLimiter` and `circuitBreaker` were omitted for brevity.
```